src/Exception/AuthorizationException.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Joindin\Api\Exception;
+
+use \Exception;
+use \Throwable;
+
+final class AuthorizationException extends Exception implements Throwable
```

Comment:
The `Throwable` interface is already implemented by `Exception`, can we remove the `implements`?